Gaijinizer patch/Troops.txt

```diff
@@ -7226,22 +7229,24 @@ Ggh...!!!#Triketos
 > BEGIN STRING
 おお！？
 > CONTEXT: Troops/170/pages/0/9/Dialogue/kazuya
-Oh! ?#MTLed
+Wow!!#Ron
 > END STRING
 
 > BEGIN STRING
 ヒノカグツチの焔が込められた火炎瓶！！
 
 サラマンダーの炎の数百倍の熱量さ！！
 > CONTEXT: Troops/170/pages/0/11/Dialogue/hyakki
-A flame bottle filled with Hinokakuguchi! !
-Hundreds of calories of Salamander's flame! !#MTLed
+It's a bottle filled with the god of fire's
+flames!!
+The flames are hundreds of times hotter than a
+salamander's!!#Ron
 > END STRING
```

Comment:
Flow:
>_**...A bottle filled with flames from the god of
fire!! The flames inside burn hundreds of times
hotter than a Salamander!!**_